utils/move_and_process.py

```diff
@@ -153,6 +154,15 @@ def get_mlt_duration(filepath):
     fps = float(m.group(1))
     return frames/fps
 
+def get_loudness(filepath):
+    cmd = ['bs1770gain', '--xml', '-pt', filepath]
+    output = subprocess.check_output(cmd, stderr=subprocess.DEVNULL)
+    output = output.decode('utf-8')
+    root = etree.fromstring(output)
+    integrated_lufs = float(root.xpath('//bs1770gain/album/track/integrated/@lufs')[0])
```

Comment:
This will throw if this is not here. Could this ever happen and be correct?

If we won't get it because there's some error, I think throwing is good, we should get the error logged and handle it in upper layers.

But are we getting `0` in cases where it won't be read? That seems to be how your code is doing this.

Reply:
It will happen when asked to measure the loudness of images, so I believe return value is more sensible than raising an exception.  But it is a close call, so I might be convinced otherwise by very few arguments. :)

Reply:
I don't care much. 

I was specifically thinking of instead of:
```
    loudness = {}
    if root.xpath('//bs1770gain/album'):
        loudness['integrated_lufs'] = float(root.xpath('//bs1770gain/album/track/integrated/@lufs')[0])
        loudness['truepeak_lufs'] = float(root.xpath('//bs1770gain/album/track/true-peak/@tpfs')[0])
    return loudness
```
Doing:
```
    try:
        integrated_lufs = float(root.xpath('//bs1770gain/album/track/integrated/@lufs')[0])
        truepeak_lufs = float(root.xpath('//bs1770gain/album/track/true-peak/@tpfs')[0])
        return integrated_lufs, truepeak_lufs
    except (IndexError, ValueError, WhatErrorLXMLXpathIsGivingOnImages) as e:
        return None, None
```
asking forgiveness instead of permission.